### PR TITLE
V3

### PR DESCRIPTION
**Pull request by ksere**
PR opened for the purpose of reviewing. **NOT TO BE MERGED**

Rel. [task](https://app.clickup.com/t/862kjndjn)

### CODE REVIEW

**Review by dpanta94 — APPROVED** [review]
LGTM!

File: main.js [hunk 1]
```diff
@@ -179,6 +180,32 @@
...
+		console.log( 'sshCommand: ' + sshCommand );
+
+		var code = await exec.exec( 'bash', [ actionPrePush ], {
+			env: {
+				PATH_DIR: localRoot,
+				GITHUB_WORKSPACE: process.env.GITHUB_WORKSPACE,
+				SSH_COMMAND: sshCommand,
+				REMOTE_ROOT: remoteRoot,
+				SSHPASS: sshPass,
+				CONSISTENCY_CHECK: ( manifest != '' ? 'true' : 'false' ),
```
Comment (msaggiorato) [inline main.js:196]: shouldn't we also check for the `consistencyCheck` variable here?

File: main.js [hunk 2]
```diff
@@ -179,6 +180,32 @@
...
+	
+		const sshCommand = shell + ' ' + remoteTarget + ' ' + shellParams.join( ' ' );
+		console.log( 'sshCommand: ' + sshCommand );
+
+		var code = await exec.exec( 'bash', [ actionPrePush ], {
+			env: {
+				PATH_DIR: localRoot,
+				GITHUB_WORKSPACE: process.env.GITHUB_WORKSPACE,
+				SSH_COMMAND: sshCommand,
+				REMOTE_ROOT: remoteRoot,
```
Comment (msaggiorato) [inline main.js:194]: Remote root is not necessarily the wp path. it's most of the time the wp-content path, but nothing guarantees it. Not sure how well WP CLI handles this.

**Review by msaggiorato — CHANGES_REQUESTED** [review]

File: main.js (hunk 2 above)
Comment (ksere) [reply main.js:194]: If you are in any folder deeper that wp root, it works properly.

**Pull request closed without merging**